website/profile/views.py

```diff
@@ -524,6 +528,10 @@ def impute_names(**kwargs):
     return auth_utils.impute_names(name)
 
 
+def help_mails(user, subscribe):
```

Comment:
The test for user choose mailing lists also tests this. Not sure how to test it otherwise since it's not an api endpoint